prow/ingress.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   annotations:
-    cert-manager.io/acme-http01-edit-in-place: "true"
+    acme.cert-manager.io/acme-http01-edit-in-place: "true"
```

Comment:
This was *blocking* the actual cert generation as `cert-manager` was trying to create an additionnal ingress and thus, the HTTP challenge was always `404`

Reply:
I think this should be `acme.cert-manager.io/http01-edit-in-place: "true"` ?
https://cert-manager.io/docs/tutorials/acme/ingress/

Reply:
yes.. I forgot to push my last change :joy: